custom_components/easee/translations/is.json

```
@@ -0,0 +1,230 @@
+{
+    "config": {
+      "abort": {
+        "already_setup": "Það er einungis hægt að vera með eina Easee tengingu"
+      },
+      "error": {
+        "auth_failure": "Ekki hægt að tengjast Easee. Athugaðu hvort notandanafnið og lykilorðið séu rétt.",
+        "connection_failure": "Ekki hægt að tengjast Easee. Reyndu aftur seinna.",
+        "no_sites": "Enginn staðsetning valin.",
```

Please add the new key below so we can include it in upcoming release
```suggestion
        "no_sites": "Enginn staðsetning valin.",
        "no_sites_in_account": "No sites found in this account",
```